README.md

```diff
@@ -48,263 +31,213 @@ You need to have the following installed:
 If you want to benchmark Java, you will also need:
   * A Java SDK 7 (`openjdk-7-jdk` package in Debian)
 
-Note that to use pinning on Linux, `cset shield` must be in a working state.
-Some Linux distributions have been known to ship with this functionality
-broken. See the `cset` tutorial for information on how to test `cset shield`:
-https://rt.wiki.kernel.org/index.php/Cpuset_Management_Utility/tutorial
 
-### Kernel arguments
+## Step 2 (Linux only): kernel and OS setup
 
-If you are using a Linux system, you will need to set a kernel arguments to
-disable Intel P-states. If your Linux bootloader is Grub, you can follow these
-steps:
+The kernel must also be
+configured to be tickless on all CPU cores except the boot core.
+\laurie{shouldn't we tell people how to do this?}
```

Comment:
I think this comment has been forgotten!

Reply:
Again, this is not the role of this repo, but instead the krun-linux-kernel repo.

Reply:
OK, I'll remove it. [This stuff was all in the original README! Don't shoot me :) ]

Reply:
ok